packages/database/src/core/RepoManager.ts

```diff
@@ -27,6 +27,9 @@ import { RepoInfo } from './RepoInfo';
 
 /** @const {string} */
 const DATABASE_URL_OPTION = 'databaseURL';
+const PROJECT_ID_OPTION = 'projectId';
```

Comment:
`projectId` is not quite what we want here, usually the namespace name is encoded directly into the `databaseURL` field. This only matters for projects with multiple databases. For instance, one of my projects, `rpb-firestore-3`, has a database instance `rpb-firestore-3-a`. I would connect to it using the SDK like
```
firebase.initializeApp({
  project: "rpb-firestore-3",
  databaseURL: "https://rpb-firestore-3-a.firebaseio.com"
})
```
or, even simpler,
```
firebase.initializeApp({ databaseURL: "https://rpb-firestore-3-a.firebaseio.com" })
```

There is a method, `parseRepoInfo`, that extracts the namespace name and stuffs it into a `RepoInfo` class. Can we use that?

Reply:
Yep, will remove this.